internal/namespaces/instance/v1/testdata/test-security-group-clear-simple.stderr.golden

```diff
@@ -0,0 +1 @@
+Missing required argument 'security-group-id'
```

Comment:
add `security-group-id` for `simple` test

another test could be done to test that error, but it should not use a golden, it should use a simple `assert()`